include/fastdds/dds/subscriber/qos/DataReaderQos.hpp

```diff
@@ -920,6 +951,9 @@ class DataReaderQos
 
     //!DataSharing configuration (Extension)
     DataSharingQosPolicy data_sharing_;
+
+    //! Thread settings for the data-sharing listener thread
+    rtps::ThreadSettings data_sharing_listener_thread_;
```

Comment:
This needs to be propagated into `rtps::ReaderAttributes`

Reply:
d278ba0fb